app/templates/components/forms/admin/content/pages-form.hbs

```diff
@@ -1,40 +1,42 @@
 <form class="ui form" {{action 'submit' data on='submit'}}>
   <div class="field">
-    <label>{{t 'Name'}}</label>
-    {{input name='name' type='text' value=data.name}}
+    <label class="required">{{t 'Name'}}</label>
```

Comment:
@kushthedude These validations have already been handled by the JS. https://github.com/fossasia/open-event-frontend/blob/b0ce33b554a1a03aa33ef9abcf2e1c37266ff39b/app/components/forms/admin/content/pages-form.js#L13-L20. 

Other than that let me what specific things you have changed?

Reply:
@pradeepgangwar I didn't make changes in validations, I have added added `Class required` to the fields which were mandatory and user were able to create page without selecting `Place` and `Language` so i have fixed that , and added a `Placeholder` for every field . 😅

Reply:
Making a field compulsory is itself a validation rule. And we do not add class to html tags to make them compulsory. See how others fields have validation rule for leaving name and other field as empty and do the same to `Place` and `Language` field. Do the changes in respective JS file.

Reply:
And `Empty Field` validation for language should work since we have defined it here. https://github.com/fossasia/open-event-frontend/blob/b0ce33b554a1a03aa33ef9abcf2e1c37266ff39b/app/components/forms/admin/content/pages-form.js#L67-L72

If it is not working then fix that part.